data/curriculum/1.json

```
@@ -0,0 +1,92 @@
+{
```

I think this should actually be an array of submodules and not an object.
```suggestion
[
```